package.json

```diff
@@ -28,12 +28,16 @@
     "bugs": {
         "url": "https://github.com/kherP/swagger-codegen-openapi/issues"
     },
-    "homepage": "https://github.com/kherP/swagger-codegen-openapi",
+    "homepage": "https://kherp.github.io/swagger-codegen-openapi/",
     "keywords": [
         "swagger",
         "codegen",
         "openapi",
         "typescript",
+        "codegen",
+        "swagger",
+        "openapi",
+        "kherp",
```

Comment:
what's this? Don't repeat the keywords, and don't add your username here

Reply:
removed lol